content/docs/command-reference/exp/run.md

```diff
@@ -109,18 +110,21 @@ execution. `dvc exp show` will mark queued experiments with an asterisk `*`.
 > Note that queuing an experiment that uses checkpoints implies `--reset`,
 > unless a `--rev` is provided (refer to the previous section).
 
-Use `dvc exp run --run-all` to process the queue. Adding `-j` (`--jobs`),
-experiment queues can be run in parallel for better performance. This creates a
-temporary workspace copy for each subprocess (in `.dvc/tmp/exps`). See also the
-`--temp` option. Queued (and temporary) experiment runs do not get applied to
-the workspace like regular ones.
+Use `dvc exp run --run-all` to process the queue. This is done outside your
+<abbr>workspace</abbr> (in a temporary dir in `.dvc/tmp/exps`), so you can
```

Comment:
Are exp queues always processed outside the workspace, even with `-j 1`? And why

Reply:
To me, "queueing" means setting up an experiment for future execution. To do that, I would expect to add to the queue, then keep making changes to my workspace without impacting the experiment that I already queued. It seems like this workflow would always require processing outside the workspace, right?

Reply:
> this workflow would always require processing outside the workspace

At `exp run --run-all` it could checkout the workspace locally at each queue time, overwriting whatever you have there and forcing you to wait for it to finish.

Reply:
> overwriting whatever you have there

It could do that, but overwriting the workspace isn't ideal, right? I know the current solution isn't perfect either, but avoiding having to overwrite the workspace seems like a big reason why exp queues are always processed outside the workspace (in addition to consistency with parallel execution and possibly remote scenarios in the future).

Reply:
Ah yeah, I agree! I'm just saying it's not necessarily obvious, so we should be explicit about the behavior I think. I guess you were just answering the Q (with a yes). Updated a bit in d8acecb4 anyway. LGTM!